beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/handler/UnreferencedMessageHandler.java

```diff
@@ -30,64 +33,86 @@
 import com.expedia.apiary.extensions.receiver.common.event.DropTableEvent;
 import com.expedia.apiary.extensions.receiver.common.event.ListenerEvent;
 
+import com.expediagroup.beekeeper.core.model.EntityHousekeepingPath;
 import com.expediagroup.beekeeper.core.model.LifecycleEventType;
 import com.expediagroup.beekeeper.scheduler.apiary.filter.EventTypeListenerEventFilter;
 import com.expediagroup.beekeeper.scheduler.apiary.filter.ListenerEventFilter;
-import com.expediagroup.beekeeper.scheduler.apiary.filter.MetadataOnlyListenerEventFilter;
+import com.expediagroup.beekeeper.scheduler.apiary.filter.LocationOnlyUpdateListenerEventFilter;
 import com.expediagroup.beekeeper.scheduler.apiary.filter.TableParameterListenerEventFilter;
 import com.expediagroup.beekeeper.scheduler.apiary.filter.WhitelistedListenerEventFilter;
-import com.expediagroup.beekeeper.scheduler.apiary.model.EventModel;
+import com.expediagroup.beekeeper.scheduler.apiary.model.UnreferencedEventModel;
 
 @Component
-public class UnreferencedMessageHandler extends MessageEventHandler {
+public class UnreferencedMessageHandler extends MessageEventHandler<EntityHousekeepingPath, UnreferencedEventModel> {
```

Comment:
Sorry, I think we should renamed this: UnreferencedLocationMessageHandler?
So make a distinction that it's dealing with Location data not metadata,